### ISSUE TITLE
[Failing Test] Conformance - GCE - master (ci-kubernetes-gce-conformance-latest)

### ISSUE THREAD

**Issue by hkamel** [kubernetes/kubernetes#95011]
**Which jobs are failing**:
>ci-kubernetes-gce-conformance-latest 

**Which test(s) are failing**:
>- Extact
>- Overall

**Since when has it been failing**:
> 09-23 06:50 PDT

**Testgrid link**:
> https://testgrid.k8s.io/sig-release-master-blocking#Conformance%20-%20GCE%20-%20master

**Reason for failure**:
> e2e.go: Extract U=https://storage.googleapis.com/kubernetes-release-dev/ci/fast R=v1.20.0-alpha.1.68+131f42d263dc6f get-kube.sh failed: error during ./get-kube.sh: exit status 56

**Anything else we need to know**:
> W0923 13:51:07.653] curl: (56) OpenSSL SSL_read: SSL_ERROR_SYSCALL, errno 104
W0923 13:51:07.656] 2020/09/23 13:51:07 process.go:155: Step './get-kube.sh' finished in 10.986531781s
W0923 13:51:07.656] 2020/09/23 13:51:07 extract_k8s.go:305: U=https://storage.googleapis.com/kubernetes-release-dev/ci/fast R=v1.20.0-alpha.1.68+131f42d263dc6f get-kube.sh failed: error during ./get-kube.sh: exit status 56
W0923 13:51:07.657] 2020/09/23 13:51:07 process.go:153: Running: ./get-kube.sh

/priority important-soon
/milestone v1.20
/sig testing
/cc @kubernetes/ci-signal

**Comment by thejoycekung** [comment]
Other failures with the same error:

- [gce-cos-master-scalability-100](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-scalability/1308759367570427904)
- [gci-gce-ingress](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-ingress/1308705510098210816)
- [gce-cos-master-reboot](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-reboot/1308782519339978752)
- [gce-cos-master-default](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce/1308833294275579904)

Also seen on 1.18:

- [skew-cluster-latest-kubectl-beta-gce](https://prow.k8s.io/view/gcs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew-stable1/1308863998728343552)

**Comment by RobertKielty** [comment]
Aggregated results for `error during ./get-kube.sh: exit status 56`

https://storage.googleapis.com/k8s-gubernator/triage/index.html?text=error%20during%20.%2Fget-kube.sh%3A%20exit%20status%2056

**Comment by eddiezane** [comment]
This appears to be an exit status used by libcurl.

```
CURLE_RECV_ERROR (56)

Failure with receiving network data.
```

```
56     Failure in receiving network data.
```

**Comment by eddiezane** [comment]
Depending on our version of curl this may be related to https://github.com/curl/curl/issues/4409/ and https://github.com/curl/curl/issues/4624.

TLDR: Potentially fixed on curl 7.68.0. Need to find out which version we have.

**Comment by eddiezane** [comment]
We're on curl 7.64.0 and the above mentioned issue wasn't introduced until 7.67.0 so most likely unrelated.

**Comment by RobertKielty** [comment]
We need to find out what is triggering these jobs as when we look at them in test-grid there is no commit associated with the jobs. Screen shot to follow ...

**Comment by eddiezane** [comment]
![image](https://user-images.githubusercontent.com/959573/94167087-16211500-fe5a-11ea-9e68-da00ff67e0aa.png)

**Comment by eddiezane** [comment]
Slack thread [here](https://kubernetes.slack.com/archives/C7J9RP96G/p1600962219002000).

The "unknown" is the data not being reported back because the job fails in the early stages - safe to disregard as the cause here.

@BenTheElder believes this is Kubernetes network flakiness with the connection being reset.

The download is being retried 3 times (confirmed in the build log) and makes decent progress with the download each time.

This appears to have started 2020-09-11: https://storage.googleapis.com/k8s-gubernator/triage/index.html?text=error%20during%20.%2Fget-kube.sh%3A%20exit%20status%2056

Not sure where to go from here.

**Comment by BenTheElder** [comment]
this doesn't appear to be an issue currently.

**Comment by thejoycekung** [comment]
@eddiezane I remember you dove a lot into this flake. It doesn't really seem to be happening currently, are you okay with closing it?

**Comment by eddiezane** [comment]
Sure thing! Looks like the retries helped.

/close
